gpMgmt/doc/gpinitsystem_help

```diff
@@ -114,12 +114,10 @@ OPTIONS
 --locale=<locale> | -n <locale>  
 
  Sets the default locale used by Greenplum Database. If not specified, 
- the LC_ALL, LC_COLLATE, or LANG environment variable of the master 
- host determines the locale. If these are not set, the default locale 
- is C (POSIX). A locale identifier consists of a language identifier 
- and a region identifier, and optionally a character set encoding. 
- For example, sv_SE is Swedish as spoken in Sweden, en_US is U.S. 
- English, and fr_CA is French Canadian. If more than one character 
+ locale set to default [en_US.UTF-8]. A locale identifier consists 
```

Comment:
Square brackets are redundant here. I don't see in those file the such style of highlighting.